src/main/java/org/spongepowered/api/world/explosion/Explosion.java

```diff
@@ -235,6 +242,17 @@ default Builder knockback(double knockback) {
             return this;
         }
 
+        /**
+         * Sets the method by which the explosion resistance of an affected block can be obtained or calculated.
+         * Expects a lambda in the form of (BlockState, Vector3i, Explosion) that returns a float.
+         *
+         * @param resistanceCalculator The lambda to use
```

Comment:
They don't have to add a lambda here. They can create a subclass in the traditional way if they prefer.

`The calculator` will suffice.